plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/githubPullRequest.ts

```diff
@@ -195,6 +201,11 @@ export const createPublishGithubPullRequestAction = ({
             description:
               'The teams that will be added as reviewers to the pull request',
           },
+          throttling: {
+            title: 'Github Throttling',
+            type: 'boolean',
+            description: 'Whether to enable Github request throttling',
```

Comment:
When would you want throttling enabled? Should we simply remove the throttling?

Reply:
Thanks for asking! I considered that since octokit v1.8.1 enables throttling by default, it should be possible to use that default. Furthermore since throttling is still a relatively new octokit default dependency/feature, I can see the feature becoming more refined/useful for the scaffolder in the future.

Reply:
What are some numbers for how well the scaffolding works with and without the throttling? I'm feeling that this really is something that shouldn't really be an option, and should just behave optimally out of the box.

Another way to look at it: if we ship this option, when should users enable/disable it? What would make a user turn it on/off.

Reply:
With throttling disabled, the `publish:github:pull-request` action takes about 10 seconds to complete, for a repo with 350 files. With throttling enabled, it's >5 minutes. However I've only tested this using our specific Github enterprise account. It might well work differently for other (types of) accounts, or it might depend on other factors that I'm unaware of. 

So variation between accounts/config might be a reason for users to enable it, and more generally you might want the option to follow upstream defaults as much as possible (i.e. the octokit defaults in this case). But I concur this is a bit of a grotty option and I'm happy to remove it as well.